packages/node-cli/src/commands/account/setup/SetupInputHandler.js

```diff
@@ -93,14 +95,17 @@ module.exports = class SetupInputHandler extends BaseInputHandler {
 			choices.push(new Separator());
 			choices.push(new Separator(NodeTranslationService.getMessage(MESSAGES.SELECT_CONFIGURED_AUTHID)));
 			authIDs.forEach((authID) => {
-				const authentication = authIDActionResult.data[authID];
-				const isDevLabel = authentication.developmentMode
-					? NodeTranslationService.getMessage(QUESTIONS_CHOICES.SELECT_AUTHID.EXISTING_AUTH_ID_DEV_URL, authentication.urls.app)
-					: '';
-				const accountInfo = `${authentication.accountInfo.companyName} [${authentication.accountInfo.roleName}]`;
+				const accountCredentials = authIDActionResult.data[authID];
+				const isNotProductionLabel =
+					!accountCredentials.urls &&
+					!accountCredentials.urls.app.match(ApplicationConstants.DOMAIN.PRODUCTION.PRODUCTION_DOMAIN_REGEX) &&
+					!accountCredentials.urls.app.match(ApplicationConstants.DOMAIN.PRODUCTION.PRODUCTION_ACCOUNT_SPECIFIC_DOMAIN_REGEX)
+						? NodeTranslationService.getMessage(QUESTIONS_CHOICES.SELECT_AUTHID.EXISTING_AUTH_ID_URL_NOT_PRODUCTION, accountCredentials.urls.app)
```

Comment:
Can we extract this check for the ternary operator to a boolean `isNotProductionUrl` to improve readability. I would also rename the label to `notProductionLabel` because I think is better to use the `isVerbPrefix` for boolean values.